tests/unit/test_loader.py

```diff
@@ -1717,8 +1717,8 @@ def test_lazyloader_refresh_file_mapping_called_with_lock_at___init__(self):
         lock_mock = MagicMock()
         lock_mock.__enter__ = MagicMock()
         self.LOADER_CLASS._refresh_file_mapping = func_mock
-        with patch("threading.RLock", MagicMock(return_value=lock_mock)):
-            loader = self.__init_loader()
+        self.LOADER_CLASS._get_lock = MagicMock(return_value=lock_mock)
+        loader = self.__init_loader()
```

Comment:
```suggestion
        with patch.object(self.LOADER_CLASS, "_get_lock", MagicMock(return_value=lock_mock)):
            loader = self.__init_loader()
```

Reply:
This is untested code, but we need to use `patch` to revert the changes after the test completes